portfolio/src/main/webapp/charts.js

```diff
@@ -28,9 +28,9 @@ function getRegionsData() {
 }
 
 /** Creates a chart and adds it to the page. */
-function drawChart(regionsData,regionsKeys) {
+function drawChart(regionsData, regionsKeys, colorAxis=['OldLace', 'SeaGreen']) {
   const data = new google.visualization.arrayToDataTable(regionsData);
-  const options = {};
+  const options = {colorAxis: {colors: colorAxis}};
```

Comment:
Do we need a nested object here?

Reply:
I might be confused with the naming of two colorAxis

Reply:
I think so because colorAxis has different attributes-- when I just put `options = {colorAxis}` it doesn't work. Maybe if I name rename the parameter to colors then I can just say `{colorAxis: {colors}}` or something https://developers.google.com/chart/interactive/docs/gallery/geochart#configuration-options

Reply:
^^ yep that works. Still a nested object but at least a little cleaner

Reply:
I see, ok yeah I think I like `{colorAxis: {colors}}`, it makes more sense to me :) Thanks for clarifying!